lib/config/options/index.ts

```diff
@@ -1499,6 +1499,38 @@ const options: RenovateOptions[] = [
     env: false,
     supportedPlatforms: ['github'],
   },
+  {
+    name: 'osvVulnerabilityAlerts',
+    description: 'Use vulnerability alerts from osv.dev',
+    type: 'boolean',
+    default: false,
+    releaseStatus: 'unpublished',
```

Comment:
do we still support this? 🤔

Reply:
This is the first I've heard of `releaseStatus`.

Do we even publish the `releaseStatus` in the docs? I don't get any hits when I put `releaseStatus` in the docs' search.

Reply:
Apparently not then 🤷 

I'll change it to alpha and remove `unpublished` as an option.